lib/cn/cn_tree.c

```diff
@@ -123,6 +129,47 @@ cn_node_free(struct cn_tree_node *tn)
     }
 }
 
+static void
+cn_node_enqueue_subspill(struct subspill *ss, struct cn_tree_node *tn)
```

Comment:
Functions that work on subspill objects should be named `cn_subspill_*()`.  I see these two that could be renamed to follow that convention:
```
cn_node_enqueue_subspill(struct subspill *ss, struct cn_tree_node *tn)
cn_node_apply_subspill(struct subspill *ss)
```